### ISSUE TITLE
ngrxFormControlState set id attribute and make label break

### ISSUE THREAD

**Issue by zhaosiyang** [MrWolfZ/ngrx-forms#86]
Sorry for my previous inactivity about issue #84 

I found that label for any input field doesn't work if that input has ngrxFormControlState.
By "doesn't work", I mean when clicking the label, the input field is not focused.

I kind of find the reason, the ngrxFormControlState will programmatically override id attribute of input, so the `for` attribute of `label` won't match the user-set id of the input.

**Comment by MrWolfZ** [comment]
Indeed, I will probably fix this behaviour by checking if the user already set an ID. In the meantime you can always just use the ID assigned by ngrx-forms in the label's `for` attribute, i.e. use something like this:

```html
<label [for]="myGroup.controls.myControl.id">My Control</label>
<input type="test" [ngrxFormControlState]="myGroup.controls.myControl" />
```

**Comment by zhaosiyang** [comment]
Is it possible to not touch the native `id` but creating some private name like `ngrx-forms-id`?

**Comment by MrWolfZ** [comment]
Setting the `id` is meant purely as a convenience feature so that users don't have to come up with their own unique ID if **ngrx-forms** can already provide one, so renaming the property wouldn't work.

However, I have just published version 2.3.6 which contains a fix that prevents setting the `id` if it is already set otherwise. This should fix your issue. If not, please let me know here and I'll re-open this issue.
